Test.cs

```diff
@@ -4,6 +4,8 @@ namespace FactorialCalculator
 {
     public class Factorial
     {
+        private branchName = "Dev"
+        private int id = 10;
```

Comment:
The `branchName` field should have an explicit type declaration, e.g., `private string branchName = "Dev";`. This ensures type safety and consistency. Also, consider renaming it to follow the camelCase convention, such as `private string BranchName = "Dev";`.